src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.scss

```diff
@@ -16,6 +16,8 @@
   @include sbb.button-reset;
 
   display: flex;
+  min-width: var(--sbb-size-icon-ui-small);
+  min-height: var(--sbb-size-icon-ui-small);
```

Comment:
I think we should apply this only on the sbb-icon element, as you can slot in any content which not necessarly has to be this min-size.